src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1054,4 +1060,4 @@ public static boolean onClientDisconnect(Connection connection, Minecraft mc, Sc
         mc.setScreen(new ModMismatchDisconnectedScreen(parent, CommonComponents.CONNECT_FAILED, message, mismatch));
         return true;
     }
-}
+}
```

Comment:
Unfortunately unable to approve until you find out whatever's causing this in your IDE and fix it :/